examples/src/main/java/nl/knaw/dans/lib/dataverse/example/DataversePublish.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2021 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.lib.dataverse.example;
+
+import nl.knaw.dans.lib.dataverse.DataverseException;
+import nl.knaw.dans.lib.dataverse.DataverseHttpResponse;
+import nl.knaw.dans.lib.dataverse.ExampleBase;
+import nl.knaw.dans.lib.dataverse.model.DataMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DataversePublish extends ExampleBase {
+
+    private static final Logger log = LoggerFactory.getLogger(DataversePublish.class);
+
+    public static void main(String[] args) throws Exception {
+        log.info("--- CREATE FIRST A test DATAVERSE IF IT DOESN'T ALREADY EXIST ---");
```

Comment:
@vesaakerman : the more I think about it, the less I like this approach of reusing the other example. Remember, these are example programs. They are intended to be clear as cristal, not necessarily very efficient. The reuse construction is a bit obscure:

* What if the DataverseException is thrown for some other reason than the test dataverse already existing?
* What if the test dataverse is already published?

I propose the following alternative to make the example both easy to use and clear to read:

Two command line arguments: `String dataverseAlias` and `boolean create`:

* If `create` is `true` first call `client.dataverse("root").create(dataverseAlias)`.
* If `create` is absent of `false` skip the creation step.

Reply:
Exactly because these are example programs, my conclusion was that we don't need further to check what was the cause of the Exception.   With the try-catch construction we get to the real point of this test, namely testing the publishing of the Dataverse.
If we would add the `create` argument, the tester would still need to know whether the Dataverse exists or not.  In my opinion it is more straightforward to simply create it, if it doesn't exist.
And if the test dataverse is already published, you get an exception, just the same way if you try to run `create` method twice.

Reply:
I am assuming this is being used by someone who knows  what the current state of the target Dataverse instance is (e.g., which dataverses exist). 

The main point is that the code is easy to read and to achieve that I want each example to be short and self-contained. I was willing to compromise with you with adding the call to optionally create the dataverse before publishing it, but that is as far as I will go. Please implement the suggestion in my previous comment or leave the creation step altogether.